models/vrp.rb

```diff
@@ -144,10 +144,10 @@ def self.check_consistency(hash)
       # shipment relation consistency
       shipment_relations = hash[:relations]&.select{ |r| r[:type] == :shipment }&.flat_map{ |r| r[:linked_ids] }.to_a
       unless shipment_relations.size == shipment_relations.uniq.size
-        raise OptimizerWrapper::UnsupportedProblemError.new(
+        raise OptimizerWrapper::DiscordantProblemError.new(
```

Comment:
This is not a discordant problem. It is a valid VRP, it is just we don't support it. 

The fix is to give the collected ids to the UnsupportedProblemError in an `Array[Array[String]]` form if I am no mistaken.